.eslintrc.json

```
@@ -0,0 +1,17 @@
+{
+  "extends": ["prettier"],
+  "env": {
+    "es2023": true,
```

Sorry, looks like I led you astray 😓 , the latest version that eslint supports is `es2022`: https://github.com/ljharb/eslintrc/commit/bc4d5304fa7f3d8f64bf3c8826549d3c2cc1088a

```suggestion
    "es2022": true,
```